app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    components = command.split()
+    if len(components) != 3:
+        raise ValueError("Invalid command format. "
+                         "Usage: cp <source_file> <destination_file>")
```

Comment:
You should also check if the first(zeroest?) component is "cp" and not something else